core/client-internal/src/main/java/alluxio/client/UfsUtils.java

```diff
@@ -42,177 +40,70 @@
+   * will continue to load metadata for each file. This method can be called repeatedly to sync
+   * the latest metadata from the under file system.
    *
-   * @param tfsRootPath the destination point in Alluxio file system to load the under file system
-   *        path onto
-   * @param ufsRootPath the source path in the under file system to be loaded
-   * @param path the path in the under file system be loaded, path.startsWith(ufsRootPath) must be
-   *        true
-   * @return the new path relative to tfsRootPath
-   */
-  private static AlluxioURI buildTFSPath(AlluxioURI tfsRootPath, AlluxioURI ufsRootPath,
-      AlluxioURI path) {
-    String filePath = path.getPath().substring(ufsRootPath.getPath().length());
-    if (filePath.isEmpty()) {
-      // retrieve the basename in ufsRootPath
-      filePath = path.getPath().substring(
-          ufsRootPath.getPath().lastIndexOf(AlluxioURI.SEPARATOR) + 1);
-    }
-    return new AlluxioURI(PathUtils.concatPath(tfsRootPath, filePath));
-  }
-
-  /**
-   * Loads files under path "ufsAddrRootPath" (excluding excludePathPrefix relative to the path) to
-   * the given tfs under a given destination path.
-   *
-   * @param tfsAddrRootPath the Alluxio file system address and path to load the src files, like
-   *        "alluxio://host:port/dest".
-   * @param ufsAddrRootPath the address and root path of the under file system, like
-   *        "hdfs://host:port/src"
-   * @param excludePaths paths to exclude from ufsRootPath, which will not be loaded in Alluxio file
-   *        system
-   * @param configuration the instance of {@link Configuration} to be used
-   * @throws IOException when an event that prevents the operation from completing is encountered
-   * @throws AlluxioException if an unexpected Alluxio error occurs
-   */
-  private static void loadUfs(AlluxioURI tfsAddrRootPath, AlluxioURI ufsAddrRootPath,
-      String excludePaths, Configuration configuration) throws IOException, AlluxioException {
-    FileSystem tfs = FileSystem.Factory.get();
-
-    PrefixList excludePathPrefix = new PrefixList(excludePaths, ";");
-
-    loadUfs(tfs, tfsAddrRootPath, ufsAddrRootPath, excludePathPrefix, configuration);
-  }
-
-  /**
-   * Loads files under path "ufsAddress/ufsRootPath" (excluding excludePathPrefix) to the given fs
-   * under the given tfsRootPath directory.
-   *
-   * @param fs the {@link FileSystem} handler created out of address like "alluxio://host:port"
-   * @param alluxioPath the destination point in Alluxio file system to load the under file system
-   *        path onto
-   * @param ufsAddrRootPath the address and root path of the under file system, like
-   *        "hdfs://host:port/dir"
-   * @param excludePathPrefix paths to exclude from ufsRootPath, which will not be registered in
-   *        Alluxio file system
+   * @param mountPointUri the destination point in Alluxio file system to load the under file system
```

Comment:
also, the parameter description says it must not already exist, but the method description suggests it is okay for it to exist